modules/i3s/src/types.ts

```diff
@@ -590,7 +590,23 @@ type TextureImage = {
   length?: number[];
 };
 
-export type Attribute = 'OBJECTID' | 'string' | 'double' | 'Int32' | string;
+export enum AttributeType {
```

Comment:
I strongly prefer new style string constants to old-school enums. This seems like a step backwards to me.

Reply:
Done